config/sources/families/tqma8mpxl.conf

```diff
@@ -0,0 +1,95 @@
+#
```

Comment:
... and filename of this family should also be changed to imx8.conf

Reply:
`ASOUND_STATE`, `SERIALCON` and probably `BOOTENV_FILE` are platform-specific. Same for `ATFSOURCE` and `ATFBRANCH` and other variables and functions

How is this supposed to work in a SoC-specific (or SoC family-specific) family config file?

Reply:
The filename in now imx8m.conf. The variables SERIALCON was moved to the board-config file.
ATFSOURCE, ATFBRANCH are defined in the imx8m familie.

```
# make some board specific adjustments
case $BOARD in
	mba8mpxl*)
		ATFSOURCE='https://github.com/tq-systems/atf' # required for ram
		ATFBRANCH="branch:TQMa8-imx_5.4.70_2.3.0"
		...
```